lib/exercism_config/version.rb

```diff
@@ -1,3 +1,3 @@
 module ExercismConfig
-  VERSION = "0.26.0"
+  VERSION = "0.26.1"
```

Comment:
Let's just keep this rolling on minor versions for now.  
```suggestion
  VERSION = "0.29.0"
```